freedom/llh_service/llh_cython.c

```diff
@@ -0,0 +1,10104 @@
+/* Generated by Cython 0.29.15 */
```

Comment:
This file will be auto generated, right? So sounds like it should not be on github

Reply:
The c file is distributed so that the package is not dependent on Cython, i.e., so that users can build the package even if they do not have Cython. This is following the Cython users' guide section on distributing cython modules.

Reply:
ah ok, sounds good then

Reply:
Actually, I might quibble with this. If you are going to put this project on python package index, then we can avoid this altogether. And I think that might be the best way to go.

So if a user is cloning the repo, he/she is a developer, and should (must) have Cython. I don't think that's an onerous thing to expect of devs, is it? If a user simply wants to use freeDOM, he/she can install via the version on the package index.

Then we don't have to have diffs in the c file every time the pyx is modified (or risk the c file not being updated even when the pyx file has been updated)...

Reply:
generating this file can be done via setup.py, right? It is just about the dependency of having cython installed (?)

Reply:
That's correct. The setup.py file currently generates the c file if cython is importable, otherwise it just compiles the existing c file. As I mentioned before, I was directly following the recommendation from the cython user guide. If you prefer to have cython as an installation dependency, then we can remove the c file from the repo.

Reply:
either's fine with me..... @jllanfranchi ?